contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3351,8 +3351,9 @@ set_special
     : SET set_on_off_option (COMMA set_on_off_option)* on_off 
     | SET STATISTICS set_statistics_keyword (COMMA set_statistics_keyword)* on_off
     | SET OFFSETS set_offsets_keyword (COMMA set_offsets_keyword)* on_off
+    | SET set_babelfish_guc LOCAL_ID
```

Comment:
please use a different name. this may be implemented with a GUC, but that is not the right name to use in the grammar. this really is a T-SQL feature. So if you need a a different grammar rule, name it  `set_option_with_variable` or something like that.